primes.rb

```diff
@@ -1,6 +1,49 @@
+
+=begin
+
+Bit lost on this at the moment, I understand what I need to do but cant seem to get it working. I am creating the $factor array
+and pushing in the square root of 2..limit and then removing any duplicates. This is to make the program more effecient incase 
+you want to find the sum of primes up to say 1,000,000 it would saving running through each number.
+
+From there I know I need to them basically just check limit.each % each of the array numbers to see if they equal 0 to rule them out
+as not prime, but seem to be brain farting it atm.
+
+I am aware there is a prime gem I could use, but seems to defeat the purpose of the exercise.
```

Comment:
Hey Simon! Thanks for the submission so far. Don't be disheartened, it's a tricky problem to solve for sure. I'd be happy to see you create a solution relying on the available library code initially. Why don't we try and get the submission complete this way, and then perhaps if you have the time you can come back and try and implement your own, non-library solution?

One quick point about your use of variables, prefixing your variables with `$` makes them globally scoped - which is generally something you'd want to avoid. We generally aim to limit the scope of any variables we declare and that means local or instance variables are used most often. Try and replace your own solution with library code from the `Prime` class and remember to limit the scope of your variables where possible. :+1: